src/urllib3/contrib/_securetransport/record.py

```diff
@@ -0,0 +1,32 @@
+import struct
+
+
+# TODO: docs
+TLS_PROTOCOL_VERSIONS = {
```

Comment:
Since we don't need to yet send more than the `UNKNOWN_CA` alert record can we condense this into a single function `build_tls_unknown_ca_alert(version)`? Then we can hardcode a lot of the `0x??` values within the function. If we decide we need to send different records we can go from there.

It'd be good to move this into `_securetransport/low_level.py` as well since it's definitely low level :P Don't think it needs a separate file yet.

Reply:
Done.